EvidenceApi.Tests/V1/Gateways/NotifyGatewayTests.cs

```diff
@@ -41,24 +41,30 @@ public void CanSendAnEvidenceRequestedSms()
             var envVar = "NOTIFY_TEMPLATE_EVIDENCE_REQUESTED_SMS";
             var resident = _fixture.Create<Resident>();
             var request = TestDataHelper.EvidenceRequest();
+            request.DocumentTypes = new List<string> { "proof-of-id", "repairs-photo" };
             request.ResidentId = resident.Id;
             request.Reason = "some-reason";
 
             var expectedTemplateId = Environment.GetEnvironmentVariable(envVar);
+            var formattedDocumentTypes = "Proof of ID,\nRepairs photo";
             var expectedParams = new Dictionary<string, object>
             {
                 {"resident_name", resident.Name},
                 {"reason", request.Reason},
                 {"magic_link", $"{_options.EvidenceRequestClientUrl}resident/{request.Id}"},
-                {"note_to_resident", request.NoteToResident}
+                {"note_to_resident", request.NoteToResident},
+                {"document_types", formattedDocumentTypes}
```

Comment:
Is this the best parameter name to use in Gov Notify? Should I simplify it to 'documents'?